test/fixtures/v2/documentRopstenValidWithDocumentStoreAndSignature.ts

```diff
@@ -0,0 +1,77 @@
+import { ProofPurpose, ProofType, SchemaId, SignedWrappedDocument, v2 } from "@govtechsg/open-attestation";
+
+interface CustomDocument extends v2.OpenAttestationDocument {
+  recipient: {
+    name: string;
+    address: {
+      street: string;
+      country: string;
+    };
+  };
+  consignment: any;
+  declaration: any;
+}
+export const documentRopstenValidWithDocumentStoreAndSignature: SignedWrappedDocument<CustomDocument> = {
+  version: SchemaId.v2,
+  schema: "tradetrust/1.0",
+  data: {
+    id: "7c89df24-c28d-4807-a1cd-abbc917f14ec:string:SGCNM21566325",
+    $template: {
+      name: "cde68ddb-5d1b-444c-a40d-708575706884:string:CERTIFICATE_OF_NON_MANIPULATION",
+      type: "16586a2a-fbb4-4afb-a7c5-3dac13b8663e:string:EMBEDDED_RENDERER",
+      url: "d1d00871-c7bb-4d8f-b3ad-2f51fac760f7:string:https://demo-cnm.openattestation.com",
+    },
+    issuers: [
+      {
+        name: "9f2c0997-4913-46e9-8b35-3b4b48ad0d93:string:DEMO STORE",
+        documentStore: "93aef98e-b830-4c45-89f8-d38fc66a68fb:string:0x8Fc57204c35fb9317D91285eF52D6b892EC08cD3",
```

Comment:
should this still be documentStore?

Reply:
is this even a valid document now??

Reply:
as discussed, this MR only about changing the skip method of the document store issuance fragment =)